pkg/gits/operations/pull_request_op.go

```diff
@@ -88,14 +88,11 @@ func (o *PullRequestOperation) CreatePullRequest(kind string, update ChangeFiles
 		filter := &gits.PullRequestFilter{
 			Labels: labels,
 		}
+		details.Labels = labels
```

Comment:
+1, that seems ways more sensible.

Any idea on why the labels were initially added in a separate call?  Seems to make no sense.